<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="pcl_grabber_viewer" name="grabber_viewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinX">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>-9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxX">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinY">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>-9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxY">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinZ">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>-9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxZ">
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Capture"/>
   <addaction name="action_Load"/>
   <addaction name="action_Test"/>
  </widget>
  <action name="action_Capture">
   <property name="text">
    <string>&amp;Capture</string>
   </property>
   <property name="toolTip">
    <string>Capture</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Test">
   <property name="text">
    <string>&amp;Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pcl_grabber_viewer</class>
   <extends>QWidget</extends>
   <header>pcl_grabber_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
